app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -0,0 +1,38 @@
+############################################################################################
+####################################### SERVICE ############################################
+############################################################################################
+
+
+ListingService = ($http, $modal) ->
```

Comment:
@kaptron On the http stuff, it might be cool if we hit a rails controller method that returns the json. It may be efficient to start building out the rails part of the application hand-in-hand with the angular part. thoughts?